packages/perseus/src/widgets/matrix.jsx

```diff
@@ -328,7 +328,14 @@ class Matrix extends React.Component<Props, State> {
                                         const updatedProps = {...inputProps};
                                         updatedProps.style = StyleSheet.create({
                                             // eslint-disable-next-line react-native/no-unused-styles
-                                            input: inputProps.style,
+                                            input: {
+                                                ...updatedProps.style,
+                                                display: "inline-block",
+                                                padding: 0,
+                                                background: outside
```

Comment:
My CSS/Style knowledge is pretty lacking.. but should this be `backgroundColor`?  I see that style key used on line 302 above when we use the same colours. 🤔

Reply:
`background` is just a shorthand for several background properties including `background-color`. I tend to favor `background` though I could see an argument for `background-color`; since we're using `background-color` above, I changed mine.